### PR TITLE
[Bug Fix] Free Memory on ONVM_MGR Shutdown

### LINKED ISSUE #119: Free malloced mem in onvm_mgr main
**Current version**
Investigate `onvm_mgr` cleanup. I don't think we ever free anything in onvm_mgr, example see https://github.com/sdnfv/openNetVM/blob/develop/onvm/onvm_mgr/main.c#L303. 

**Expected code:**
Call free or rte_free depending on memory type.

### PR DESCRIPTION
**Pull request by koolzz**
Before we had a tiny memleak within onvm_mgr, this pr frees allocated memory and resolves that.

## Summary:
Free memory from within the threads themselves, assures that structs are unused by the time they are freed.

**Usage:**
Nothing new run onvm_mgr as usual.

| This PR includes         |          |
| ------------------------ | -------- |
| Resolves issues          |  Fixes #119 
| Breaking API changes     |
| Internal API changes     |
| Usability improvements   |  
| Bug fixes                | 🚰 
| New functionality        |
| New NF/onvm_mgr args     | 
| Changes to starting NFs  |  
| Dependency updates       | 
| Web stats updates        | 

## Merging notes:
 - Dependencies: None  

**TODO before merging :**
 - [x] PR is ready for review

## Test Plan:
Check that it doesn't segfault both for shared core and normal mode. Also check if there is anything else allocated within `onvm_mgr/main.c`

## Review: 
@dennisafa @kevindweb this is pretty tiny so just review whoever has time (one reviewer is enough)

### CODE REVIEW

**Comment by onvm** [comment]
> In response to PR creation

## CI Message

Your results will arrive shortly

File: onvm/onvm_mgr/main.c [hunk 1]
```diff
@@ -253,6 +255,9 @@ tx_thread_main(void *arg) {
 
         RTE_LOG(INFO, APP, "Core %d: TX thread done\n", rte_lcore_id());
 
+        free(tx_mgr->tx_thread_info);
```
Comment (dennisafa) [inline onvm/onvm_mgr/main.c]: Should we free(tx_mgr->tx_thread_info->port_tx_bufs) as well?

Comment (koolzz) [reply onvm/onvm_mgr/main.c]: We definitely should, good catch

Comment (koolzz) [reply onvm/onvm_mgr/main.c]: @dennisafa fixed

**Comment by koolzz** [comment]
@onvm sanity check

**Comment by onvm** [comment]
> @onvm sanity check

## CI Message

Your results will arrive shortly

**Pull request merged**